noxfile.py

```diff
@@ -30,7 +30,7 @@
     BLACK_PATHS.append("samples")
 
 
-@nox.session(python="3.7")
+@nox.session(python=["3.7", "3.8"])
```

Comment:
Thank you for pushing these changes!

The things is that noxfiles are auto generated, you can see the disclaimer on the line 17.
That means we should not do any changes manually, as they will be reverted on the next code generation.
On the other hand, adding such a changes into templates from which noxfiles will be generated, means that all Google Cloud clients will be affected. I suppose, let's wait for responsible googler to approve these issues first, as it sounds like a weighty decision.

Reply:
I did not see this comment (sorry, I skimmed the file and the made a minimal change so that I could actually run the tests).

I am happy to wait for an answer from someone else. I can also remove this specific commit from the PR if you prefer (though once I do, I will be unable to rerun the tests if I am requested to make changes to other parts of this PR).